setup.py

```diff
@@ -110,6 +112,16 @@
   DEFINE_MACROS += (('PyMODINIT_FUNC', pymodinit),)
 
 
+# By default, Python3 distutils enforces compatibility of
+# plugins with the OSX version Python3 was built with.
```

Comment:
By plugins do you mean extensions (e.g. `.so` and/or `.pyd` files)? Or the distutils custom commands? I'm guessing the former but just want to make sure.